ext/src/main/java/io/burt/kafka/clients/KafkaClientsLibrary.java

```diff
@@ -58,6 +59,8 @@ static RubyClass mapErrorClass(Ruby runtime, Throwable t) {
       String javaName = t.getClass().getSimpleName();
       String rubyName = javaName.substring(0, javaName.length() - 9) + "Error";
       return (RubyClass) runtime.getClassFromPath(String.format("Kafka::Clients::%s", rubyName));
+    } else if (t instanceof TimeoutException) {
+      return (RubyClass) runtime.getModule("Timeout").getConstant("Error");
```

Comment:
This feels like a better solution: https://github.com/burtcorp/kafka-clients-jruby/pull/5/commits/ac4a17ea26cccfa4248bd1f6f0d18e2c6bac302a